### DISCUSSION TITLE
How to find which plot won block?

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#3309] [General]
I read the opinion that won plots should be "replotted".

Is it correct?

How to find won plots?

**Comment by Akaino** [comment]
No. 
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#i-have-heard-that-its-recommended-that-a-winning-plot-be-deleted-on-mainnet

**Reply by ghost** [reply]
> This attack is much more difficult and thus less of a risk in new consensus and thus we only recommend deleting and re-plotting a plot to farmers with in excess of 1PB of farm size.

Let's say we do not have such big farm. Though.. how to find won plot? :)

**Comment by rijulluman** [comment]
I found the answer here
https://thechiafarmer.com/2021/04/30/which-plot-won/?amp
Hope it helped :)

1. Go to the Chia Explorer website. (https://www.chiaexplorer.com/)
2. On this webpage, change the search type to “Address“.
3. Enter you Wallet Receive Address. This is located in the Wallet tab of the Chia Client. Copy and paste the address. Don’t hand type it.
4. After it searches, you should see your address in green font. Click it to open the details of your wallet.
5. At the new page, scroll down to see your farmer rewards. Here, you should be able to see the “height” that the reward was obtained. Click on the height number. It doesn’t matter if it was for the 0.25 or 1.75 reward.
6. Next, Find the value for proof_of_space.plot_public_key. This is the Public Key of your plot.
7. Now go to the Plot tab in the Chia Client and find the plot with that Public Key.

**Reply by ghost** [reply]
> At the new page, scroll down to see your farmer rewards.

I missing that part. Maybe because it is already spent.

**Reply by rijulluman** [reply]
For example here is a random block:
https://www.chiaexplorer.com/blockchain/block/1622723724
Just search "proof_of_space.plot_public_key" and you will find it on that page

**Reply by rijulluman** [reply]
![image](https://user-images.githubusercontent.com/16553348/120647793-d4585d80-c42f-11eb-9781-f03d1c572d09.png)

**Comment by joelrb** [comment]
> I read the opinion that won plots should be "replotted".
> 
> Is it correct?
> 
> How to find won plots?

Read Wiki, FAQ, etc. Your questions will be answered.

Plots have many (hash) cells inside so if one wins there is still many left that can win. In other words, you do not need to delete a plot if it wins you coins. Plots can keep earning you coins. Plots are estimated to last over 5 years.
https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide

**Comment by sharjeelaziz** [comment]
If you are on Linux you can try RPC calls to list plot public keys and find the plot that won. Refer to the following gist:

https://gist.github.com/sharjeelaziz/1beb83f571f621e4f2d9b36a404b8713
